cassandra_nodetool/datadog_checks/cassandra_nodetool/data/conf.yaml.example

```diff
@@ -50,11 +59,31 @@ instances:
     #
     # ssl: false
 
-    ## @param tags - list of key:value element - optional
-    ## List of tags to attach to every metric, event and service check emitted by this integration.
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this instance.
     ##
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
     #
     # tags:
     #   - <KEY_1>:<VALUE_1>
     #   - <KEY_2>:<VALUE_2>
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
```

Comment:
```suggestion
    ## Attaches the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
```

Reply:
This is from the base template, this file is generated